velox/docs/configs.rst

```diff
@@ -129,6 +129,14 @@ Generic Configuration
      - 0
      - If it is not zero, specifies the time limit that a driver can continuously
        run on a thread before yield. If it is zero, then it no limit.
+   * - prefixsort_max_normalized_key_size
```

Comment:
It is useful to indicate units in the configuration property name. Let's rename to 

prefixsort_normalized_key_max_bytes